zerver/management/commands/soft_deactivate_users.py

```diff
@@ -30,7 +27,7 @@ def get_users_from_emails(
             f"{user_emails_not_found}\n\n"
             "Check if they are correct.",
         )
-    return users
+    return list(users)
```

Comment:
I think this belongs in the first line of this function, so we're not doing the `len()` separate database query.